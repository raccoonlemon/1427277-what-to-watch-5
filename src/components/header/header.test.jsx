import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";
import Header from "./header";

const renderer = new ShallowRenderer();

describe(`<Header> renders correctly`, () => {

  it(`with user menu`, () => {
    renderer.render(<Header needToShowUserMenu={true}/>);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });

  it(`without user menu`, () => {
    renderer.render(<Header needToShowUserMenu={false}/>);
    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });

});
