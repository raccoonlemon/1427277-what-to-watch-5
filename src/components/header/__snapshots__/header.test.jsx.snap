// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders correctly with user menu 1`] = `
<header
  className="page-header "
>
  <Logo
    className=""
  />
  <Memo(Connect(UserMenu)) />
</header>
`;

exports[`<Header> renders correctly without user menu 1`] = `
<header
  className="page-header "
>
  <Logo
    className=""
  />
</header>
`;
